qulice-pmd/src/test/java/com/qulice/pmd/PMDValidatorTest.java

```diff
@@ -91,6 +91,12 @@
         "Avoid using Plain JUnit assertions";
 
     /**
+     * Template for using non transient fields.
+     */
+    private static final String NON_TRANSIENT =
```

Comment:
@vkuchyn in most cases there is no point in making single use constants (there are cases where they add readability, but not here)